pytest.ini

```diff
@@ -26,6 +26,13 @@ filterwarnings =
     ignore: `alltrue` is deprecated as of NumPy 1.25.0
     ignore: `sometrue` is deprecated as of NumPy 1.25.0
 
+    # matplotlib and acis_thermal_check
+    ignore: Conversion of an array with ndim > 0 to a scalar is deprecated
+
+    # acis_thermal_check
+    ignore: RuntimeWarning: invalid value encountered in cast
```

Comment:
And it seems like you'd want me to cut it from here and add it to the acis_thermal_check post check logs.

Reply:
right, I guess in post check logs is ok, but I would remove these two from here. Note that the ignores above have a comment saying where they occur, and it's in core packages. Before adding them here, I checked that they didn't occur in our code (in the tested parts, of course)

I think that we should fix the warnings that happen in our code, and not ignore them.

Reply:
Even post-check-logs is kind of dubious if the warning is issued by our code.

Reply:
I think we need to limit the good-practices-on-our-code to code we actually control, which doesn't include the acis packages.

Reply:
ah, ok. I was under the assumption that acis packages were more or less in our zone.

Reply:
I have thought about it some more, and I still do not agree with having these warnings ignored. These are real warnings that can happen in our code, and ignoring them will make us blind to them, and the cause of the warnings can cause real problems at some point.

Reply:
I'm with @javierggt that these seem like potential real problems. Maybe @jzuhone can comment? Even the older ERFA warnings make me wonder what's going on.

Reply:
Right, I already agreed to take it out of the pytest.ini.

Reply:
Which is where you are commenting.  With regard to adding an "invalid value found in cast" to the post check logs, I updated the issue https://github.com/acisops/acis_thermal_check/issues/69 from January to include that warning too.

Reply:
Hi all, 

Yes, I have seen these also. 

The ERFA one is benign for sure because it comes from the fact that we have a place or two in the ACIS code where we use placeholder years far in the future. 

I am not as sure about the casting warning--I first saw it a few days ago and have it on my to-do list to investigate. Model results look unchanged, so I strongly suspect it is also benign.